stable/clamav/templates/deployment.yaml

```diff
@@ -23,6 +23,10 @@ spec:
         - name: {{ .Chart.Name }}
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
+          volumeMounts:
```

Comment:
Can you add a condition here, like {{- if .Values.freshclamConfig -}} ? (lines 25-29)
in case this is not set then we don't try to mount a volume from a  non existing configmap (as the config map have the condition)
the same in line 44-47

Reply:
Thanks for the suggestion, I added both conditions.